src/tests/unit/tests/DetailsView/components/assessment-instance-table.test.tsx

```diff
@@ -41,302 +43,143 @@ describe('AssessmentInstanceTableTest', () => {
+                    key: 'col1',
+                    name: 'col1',
+                    fieldName: 'fieldName',
+                    minWidth: 0,
                 },
-            },
-        };
-        expectedMessage = {
-            message: <div className="no-failure-view">No failing instances</div>,
-            instanceCount: 1,
-        } as DefaultMessageInterface;
+            ];
+
+            assessmentInstanceTableHandlerMock
+                .setup(a => a.createAssessmentInstanceTableItems(props.instancesMap, props.assessmentNavState, props.hasVisualHelper))
+                .returns(() => items)
+                .verifiable(Times.once());
+
+            assessmentInstanceTableHandlerMock
+                .setup(a => a.getColumnConfigs(props.instancesMap, props.assessmentNavState, props.hasVisualHelper))
+                .returns(() => cols)
+                .verifiable(Times.once());
+        });
+
+        it('renders per snapshot', () => {
+            const testSubject = shallow(<AssessmentInstanceTable {...props} />);
+            expect(testSubject.getElement()).toMatchSnapshot();
+        });
+
+        it('renders per snapshot with null header', () => {
+            const testSubject = shallow(<AssessmentInstanceTable {...props} renderInstanceTableHeader={() => null} />);
+            expect(testSubject.getElement()).toMatchSnapshot();
+        });
+
+        it('prefers rendering with getDefaultMessage if non-null', () => {
+            defaultMessage = {
+                message: <>Message from getDefaultMessage</>,
+                instanceCount: 1,
+            } as DefaultMessageInterface;
+
+            const testSubject = shallow(<AssessmentInstanceTable {...props} />);
+            expect(testSubject.getElement()).toEqual(defaultMessage.message);
+
+            getDefaultMessageMock.verifyAll();
+        });
+
+        it("delegates the underlying list's onItemInvoked to the handler's updateFocusedTarget", () => {
+            const fakeItem = { instance: { target: ['fake-instance-target-0'] } };
+            assessmentInstanceTableHandlerMock.setup(a => a.updateFocusedTarget(fakeItem.instance.target)).verifiable(Times.once());
+
+            const testSubject = mount(<AssessmentInstanceTable {...props} />);
+            testSubject.find(DetailsList).prop('onItemInvoked')(fakeItem);
+
+            assessmentInstanceTableHandlerMock.verifyAll();
+        });
+
+        describe('"Pass all unmarked instances" button', () => {
+            const passUnmarkedInstancesButtonSelector = `button[data-automation-id="${passUnmarkedInstancesButtonAutomationId}"]`;
+            it('is enabled if there is an instance with unknown status', () => {
+                testStepResults[selectedTestStep] = { status: ManualTestStatus.UNKNOWN };
+
+                const testSubject = mount(<AssessmentInstanceTable {...props} />);
+                expect(testSubject.find(passUnmarkedInstancesButtonSelector).prop('disabled')).toBeFalsy();
```

Comment:
We tend to favor `.toBe(false)` instead of `toBeFalsy()`, right? 
Is the `disabled` prop not present in this case? If that's the case I guess you could assert it as such more explicitly.